lib/rubocop/cop/style/attr.rb

```diff
@@ -21,6 +21,7 @@ class Attr < Cop
 
         def on_send(node)
           return unless node.command?(:attr) && node.arguments?
+          return if node.parent && !node.parent.class_type?
```

Comment:
So, the idea here is to check you're not calling the method in the class/module body, right? Might be a good idea to add a note explaining it.

Reply:
I believe this will create some false negatives, for example in `#class_eval` blocks?

Reply:
Thanks I missed that! I just made a change to consider `class_eval` & `module_eval`.